windows/install/install_deps.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+export DISTRO_ID="UNKNOWN"
+export DISTRO_VERSION="UNKNOWN"
+
+to_upper () {
+  echo $@ | tr [a-z] [A-Z]
+}
+
+detect_msys2_shell () {
+    if [ "$MSYSTEM" = "MINGW64" ] || [ "$MSYSTEM" = "CLANG64" ] || [ "$MSYSTEM" = "CLANGARM64" ]; then
+    	export DISTRO_ID=$MSYSTEM
+    else
+        echo "Unable to detect Windows version..."
+        echo "Please run in CLANG terminal"
+    fi
+}
+
+install_deps () {
+  case $1 in
+	MINGW64 )
+	  echo Installing depencies with $1 method
+	  echo You are about to install the dependencies using the following command:
+	  echo   pacman -S --needed --noconfirm --disable-download-timeout mingw-w64-x86_64-clang mingw-w64-x86_64-gcc mingw-w64-x86_64-gdb mingw-w64-x86_64-cmake mingw-w64-x86_64-SDL2 mingw-w64-x86_64-SDL2_gfx mingw-w64-x86_64-SDL2_mixer mingw-w64-x86_64-SDL2_image mingw-w64-x86_64-SDL2_ttf mingw-w64-x86_64-SDL2_net mingw-w64-x86_64-civetweb
+	  pacman -S --needed mingw-w64-x86_64-clang mingw-w64-x86_64-gcc mingw-w64-x86_64-gdb mingw-w64-x86_64-cmake mingw-w64-x86_64-SDL2 mingw-w64-x86_64-SDL2_gfx mingw-w64-x86_64-SDL2_mixer mingw-w64-x86_64-SDL2_image mingw-w64-x86_64-SDL2_ttf mingw-w64-x86_64-SDL2_net mingw-w64-x86_64-civetweb
+	  ;;
+  CLANG64 )
+	  echo Installing depencies with $1 method
+	  echo You are about to install the dependencies using the following command:
+	  pacman -S --needed mingw-w64-clang-x86_64-clang mingw-w64-clang-x86_64-gcc mingw-w64-clang-x86_64-gdb mingw-w64-clang-x86_64-cmake mingw-w64-clang-x86_64-SDL2 mingw-w64-clang-x86_64-SDL2_gfx mingw-w64-clang-x86_64-SDL2_mixer mingw-w64-clang-x86_64-SDL2_image mingw-w64-clang-x86_64-SDL2_ttf mingw-w64-clang-x86_64-SDL2_net mingw-w64-clang-x86_64-civetweb
+	  ;;
+	# CLANGARM64 )
```

Comment:
Have we been able to test this? Any chance we can get this into this PR?

Reply:
I've tested this and the pacman command works. Just need to sort out the testing to ensure issues are not being hidden by x64 emulation happening on my VM that wouldn't be present on a regular Windows ARM laptop.